adapter/integration-test/src/adapter-datasource-trigger.test.js

```diff
@@ -17,7 +17,7 @@ const EXECUTION_TOPIC = process.env.AMQP_IMPORT_TOPIC
 const EXECUTION_SUCCESS_TOPIC = process.env.AMQP_IMPORT_SUCCESS_TOPIC
 const EXECUTION_FAILED_TOPIC = process.env.AMQP_IMPORT_FAILED_TOPIC
 
-const STARTUP_DELAY = 2000
+const STARTUP_DELAY = 5000
```

Comment:
Why did you raise the delay? Did the adapter it still fail?

Reply:
Yes, adapter IT still failed, see [this run](https://github.com/jvalue/open-data-service/runs/1086556642) from commit 2b2d219. I have also moved the sleep before connecting to amqp. According to the logs of the failing test the `Connected to AMQP on host ...` message from the [`connectAmqp()`](https://github.com/jvalue/open-data-service/blob/30a7be612be62bf85a99164df5c4ee380188699b/adapter/integration-test/src/adapter-datasource-trigger.test.js#L185) method was never outputed to the logs.